fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/CacheDictManager.java

```diff
@@ -133,7 +134,7 @@ protected static CacheDictManager getInstance() {
             return Optional.empty();
         }
         int dictSize = tGlobalDict.getIdsSize();
-        ColumnIdentifier columnIdentifier = new ColumnIdentifier(statisticData.tableId, statisticData.columnName);
```

Comment:
what’s in the statisticData.tableId and statisticData.columnName? what the different between  statisticData.tableId and tableId now

Reply:
```
java.util.concurrent.CompletionException: java.lang.NullPointerException
        at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:273)
        at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:280)
        at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1592)
        at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1582)
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
        at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
        at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
        at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
Caused by: java.lang.NullPointerException
        at com.starrocks.sql.optimizer.base.ColumnIdentifier.equals(ColumnIdentifier.java:54)
        at java.util.concurrent.ConcurrentHashMap.putVal(ConcurrentHashMap.java:1035)
        at java.util.concurrent.ConcurrentHashMap.put(ConcurrentHashMap.java:1006)
        at java.util.Collections$SetFromMap.add(Collections.java:5461)
        at com.starrocks.sql.optimizer.statistics.CacheDictManager.deserializeColumnDict(CacheDictManager.java:138)
        at com.starrocks.sql.optimizer.statistics.CacheDictManager.access$000(CacheDictManager.java:34)
        at com.starrocks.sql.optimizer.statistics.CacheDictManager$1.lambda$asyncLoad$0(CacheDictManager.java:63)
        at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1590)
        ... 5 more
```
statisticData.columnName was null. because we don't select any column name from BE